### ISSUE TITLE
Wistia embed support?

### ISSUE THREAD

**Issue by litzinger** [spicywebau/craft-embedded-assets#237]
### What would you like to see added to or changed in Embedded Assets, and why?

Hello, I'm using your plugin for the first time and wondering if you have plans to add support for Wistia embeds?

**Comment by ttempleton** [comment]
I'll look at adding proper support with `getIframeCode()`, `getVideoId()` embedded asset methods for Wistia when I get a chance - but for now, you can allow Wistia embeds by setting the following in your project's `config/embeddedassets.php` file:

```php
<?php

return [
    'extraWhitelist' => [
        'wistia.com',
        'wistia.net',
    ],
];
```

**Comment by litzinger** [comment]
Thanks for sharing that. Are you suggesting that adding the config should bring a basic level of support for Wistia without any additional code changes or you have to add additional changes to fully support it? Currently with that config added I still get the following error:

<img width="770" alt="Screenshot 2023-07-17 at 8 04 06 AM" src="https://github.com/spicywebau/craft-embedded-assets/assets/53965/8ee64fb0-e79a-4b31-8d76-efd518efd306">

**Comment by ttempleton** [comment]
Yes, that should give you basic Wistia support - and it's working for me:
![Screenshot 2023-07-19 at 18-00-28 Homepage - Craft 4](https://github.com/spicywebau/craft-embedded-assets/assets/19421878/f044743d-53cf-4bce-9cb9-845d2d7caf69)

Do the Craft logs show any related error information? Does it still occur in another environment or a different Craft project running on a different server?

**Comment by litzinger** [comment]
I just saw this in my console :/

```Refused to load https://nsf-dev.hcog.io/index.php?p=admin%2Factions%2Fembeddedassets%2Factions%2Fpreview&site=US_en&showContent=1&callback=embeddedassetsgeq56bb913h6&url=https%3A%2F%2Ffast.wistia.net%2Fembed%2Fiframe%2Fe4a27b9710 because it does not appear in the frame-ancestors directive of the Content Security Policy.```

**Comment by litzinger** [comment]
I figured it out, it is a local environment issue with the CSP.

**Comment by ttempleton** [comment]
Just released 3.1.6, which adds `wistia.com` and `wistia.net` to the default plugin whitelist, and adds support for Wistia to embedded assets' `getVideoId()`, `getIframeSrc()`, and `getIframeCode()` methods.
